document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/mapping/mappers/CompanyReportMapperDecorator.java

```diff
@@ -38,6 +44,12 @@ public CompanyReport mapCompanyReport(CompanyReportApiData companyReportApiData)
             if (companyReportApiData.getCompanyProfileApi().getPreviousCompanyNames() != null) {
                 companyReport.setPreviousNames(setPreviousNames(companyReportApiData.getCompanyProfileApi().getPreviousCompanyNames()));
             }
+
+            if(companyReportApiData.getCompanyProfileApi().getAccounts() != null) {
+                if (companyReportApiData.getCompanyProfileApi().getAccounts() != null) {
```

Comment:
Looks as though this line could be removed as it's doing the same as the above line.